daprdocs/content/en/developing-applications/building-blocks/actors/actor-reentrancy.md

```diff
@@ -21,24 +17,10 @@ ActorA -> Actor B -> Actor A
 With reentrancy, there can be more complex actor calls without sacrificing the single-threaded behavior of virtual actors.
 
 ## Enabling actor reentrancy
-Actor reentrancy is currently in preview, so enabling it is a two step process.
-
-### Preview feature configuration
-Before using reentrancy, the feature must be enabled in Dapr. For more information on preview configurations, see [the full guide on opting into preview features in Dapr]({{< ref preview-features.md >}}). Below is an example of the configuration for actor reentrancy:
-
-```yaml
-apiVersion: dapr.io/v1alpha1
-kind: Configuration
-metadata:
-  name: reentrantconfig
-spec:
-  features:
-    - name: Actor.Reentrancy
-      enabled: true
-```
+Actor reentrancy is a one step process.
 
 ### Actor runtime configuration
```

Comment:
Instead of having a header with just the one line in it, I'd have just one header like `Enable Actor Reentrancy with Actor Configuration`.